verticapy/vdataframe.py

```diff
@@ -2541,7 +2524,7 @@ def aggregate(
                     for elem in agg
                 ]
                 query = (
```

Comment:
there is query defined just before this.

Reply:
yes but this one is using it again.